webpack/webpack5-localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -231,9 +232,11 @@ export class LocalizationPlugin implements WebpackPluginInstance {
 
                   const noLocaleExpression: string = JSON.stringify(this._noStringsLocaleName);
 
-                  return `" + (${JSON.stringify(chunkMapping)}[chunkId]?${
-                    isLocalizedSmaller ? localeExpression : noLocaleExpression
-                  }:${isLocalizedSmaller ? noLocaleExpression : localeExpression}) + "`;
+                  return this._formatLocaleForFilename(
+                    `" + (${JSON.stringify(chunkMapping)}[chunkId]?${
```

Comment:
@dmichon-msft think this is incorrect usage here. If I understand correctly is this the path getting output into the chunk referenced code? Not sure what this logic branch does.